api/plugins/module_utils/api_client.py

```diff
@@ -641,7 +641,9 @@ def deploy_target_config_delete_mutation(self, project_id, config_id):
 		    input: { project: %d, id: %d })""" % (config_id, project_id, config_id)
 
     def make_api_call(self, payload):
-        # display.v("API call payload: %s" % payload)
+        verbosity = int(os.getenv('ANSIBLE_VERBOSITY'))
+        if verbosity >= 1:
+            display.v("API call payload: %s" % payload)
```

Comment:
`display.v` already works for the condition you're checking.

E.g, if you wanted to log for higher verbosity levels, you'd do display.vv, display.vvv, display.vvvv

```suggestion
        display.v("API call payload: %s" % payload)
```